pages/help.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type { NextPage } from 'next'
+import Layout from '../components/layout/Layout'
+import CSVTable from '../components/CSVTable'
+import { ThemeProvider, Typography, Divider } from '@mui/material'
+import { StyledPageContainer } from '../styles/common'
+import { TextContainer } from '../pageStyles/home.styles'
+import { theme } from '../styles/theme'
+
+const Help: NextPage = () => {
+  return (
+        <Layout>
+        <ThemeProvider theme={theme}>
+          <StyledPageContainer>
+            <Typography variant="h3">Help</Typography>
+            <Typography variant="body1">
+            <br />
+              Use this emailsender to quickly send the same email to multiple recipients at once
```

Comment:
Let's change 'emailsender' to email sender since we're not naming a variable 
/ file - we're assuming this is a instruction guide for non cs people